test/main-test.js

```diff
@@ -0,0 +1,31 @@
+/* global window */
+
+var file, testFiles = [];
+
+for (file in window.__karma__.files) {
+    if (/spec.*\.js$/.test(file)) {
+        testFiles.push(file);
+    }
+    if (/.*\.json\.js$/.test(file)) {
```

Comment:
It seems this all could be handled with one regex, sth like: `/(?:spec.*\.js|.*\.json\.js)$/`.